commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/generators/DraftBuilderGenerator.java

```diff
@@ -128,6 +130,117 @@ private void createCopyFactoryMethods(final TypeElement resourceDraftValueType,
         }
     }
 
+    private boolean isListProperty(PropertyGenModel propertyGenModel){
+        final boolean isParmetrizedList = (propertyGenModel.getType() instanceof ParameterizedTypeName) && ((ParameterizedTypeName)propertyGenModel.getType()).rawType.reflectionName().equals(List.class.getTypeName());
+        final boolean isNonParmetrizedList = (propertyGenModel.getType() instanceof ClassName) && ((ClassName)propertyGenModel.getType()).reflectionName().equals(List.class.getTypeName());
+
+        return isNonParmetrizedList || isParmetrizedList;
+    }
+
+    private List<MethodSpec> createPlusLists(final TypeElement resourceDraftValueType,final List<PropertyGenModel> propertyGenModels){
+
+        return propertyGenModels.stream()
+                .filter(this::isListProperty)
+                .map(propertyGenModel -> createPlusListMethodSpec(resourceDraftValueType,propertyGenModel))
+                .collect(Collectors.toList());
+
+    }
+
+    private MethodSpec createPlusListMethodSpec(final TypeElement resourceDraftValueType, PropertyGenModel property){
+        final String methodName = "plus" + StringUtils.capitalize(property.getName());
+        MethodSpec.Builder builder = MethodSpec
+                .methodBuilder(methodName)
+                .addModifiers(Modifier.PUBLIC)
+                .addJavadoc("Concatenate {@code $L} parameter to the {@code $L} List property of this builder.\n\n", property.getName(), property.getName())
+                .addJavadoc("@param $L the value for $L\n", property.getJavaIdentifier(), property.getJavadocLinkTag())
+                .addJavadoc("@return this builder\n")
+                .addParameter(createParameter(property,property.getType(),false))
+                .addCode("this.$L =  $T.listOf($T.ofNullable(this.$L).orElseGet($T::new), $L);\n", property.getName(), SphereInternalUtils.class,Optional.class, property.getName(), ArrayList.class,property.getName());
+        addBuilderMethodReturn(resourceDraftValueType,builder);
+        return builder.build();
+    }
+
+    private List<MethodSpec> createPlusTolListElementsMethodSpec(final TypeElement resourceDraftValueType, final List<PropertyGenModel> propertyGenModels){
+
+        return propertyGenModels.stream()
+                .filter(this::isListProperty)
+                .map(propertyGenModel -> createPlusToListElementMethodSpec(resourceDraftValueType,propertyGenModel))
+                .collect(Collectors.toList());
+
+    }
+
+    private MethodSpec createPlusToListElementMethodSpec(final TypeElement resourceDraftValueType, PropertyGenModel property){
```

Comment:
The `final` modifier is missing on the last parameter.

Reply:
good catch 👍